test/component_fixtures/hello_world/hello-world.js

```diff
@@ -0,0 +1,15 @@
+export function greet(who) {
```

Comment:
more a FYI because I think we shouldn't tackle this in this PR: I'd love to change these js modules into Rust modules.

not so much because I don't like JS, but to reduce the number of different stacks we have to support in this Repo :)  I'm also hoping to reduce test runtime, but thats a side goal :)

Reply:
Yeah I agree, I was just too lazy to do it :) This one would probably be easy to do in rust, the http wasi example might be a little more effort.